api/tests/unit/domain/usecases/start-writing-campaign-assessment-results-to-stream_test.js

```diff
@@ -478,7 +480,8 @@ describe('Unit | Domain | Use Cases | start-writing-campaign-assessment-results
 
       it('should return the header in CSV styles with all competence, domain and skills', async () => {
```

Comment:
"should return the header **and the data** in CSV styles with all competence, domain and skills"